gobblin-utility/src/main/java/gobblin/util/ConfigUtils.java

```diff
@@ -462,4 +464,34 @@ public static boolean verifySubset(Config superConfig, Config subConfig) {
     }
     return true;
   }
+
+  /**
+   * Resolves encrypted config value(s) by considering on the path with "encConfigPath" as encrypted.
+   * It will use Password manager via given config. Thus, convention of PasswordManager need to be followed in order to be decrypted.
+   * Note that "encConfigPath" path will be removed from the config key, leaving child path on the config key.
+   * e.g:
+   *  encConfigPath = enc.conf
+   *  - Before : { enc.conf.secret_key : ENC(rOF43721f0pZqAXg#63a) }
+   *  - After  : { secret_key : decrypted_val }
+   *
+   * @param config
+   * @param encConfigPath
+   * @return
+   */
+  public static Config resolveEncrypted(Config config, Optional<String> encConfigPath) {
```

Comment:
Sure, just make it clear that passing `Optional.absent()` will not change anything, as opposed to attempting to decrypt everything.